app/views/admin/dashboard/index.html.haml

```diff
@@ -57,7 +57,7 @@
       %h4= t 'admin.dashboard.software'
       %ul
         %li
-          Mastodon
+          Florence-Mastodon
```

Comment:
```suggestion
          Florence
```

Reply:
My only worry is admins being confused when they see "Florence-Mastodon" and then "Mastodon" right underneath it.

Reply:
Any reason it has to be "Florence-Mastodon" and not just "Florence"?

Reply:
I guess it's me basically trying to avoid conflating "Florence = Florence-Mastodon" even though in this case it's unambiguous and doesn't matter.

Reply:
Honestly, I'm just gonna accept this and say it's probably fine. We can change it later.